test-suite failure on Debian 9: cppd      FAIL     0.01 s (killed by signal 11 SIGSEGV)

Trying to run the test-suite on Debian 9, the following test fails:
```
2/2 cppd                                    FAIL     0.01 s (killed by signal 11 SIGSEGV)
```
might be related to
```
Code:
 
        #include <linux/if.h>
Compiler stdout:
 
Compiler stderr:
 In file included from /tmp/tmpp25lvr20/testfile.c:2:0:
/usr/include/linux/if.h:234:19: error: field ‘ifru_addr’ has incomplete type
   struct sockaddr ifru_addr;
                   ^~~~~~~~~
/usr/include/linux/if.h:235:19: error: field ‘ifru_dstaddr’ has incomplete type
   struct sockaddr ifru_dstaddr;
                   ^~~~~~~~~~~~
/usr/include/linux/if.h:236:19: error: field ‘ifru_broadaddr’ has incomplete type
   struct sockaddr ifru_broadaddr;
                   ^~~~~~~~~~~~~~
/usr/include/linux/if.h:237:19: error: field ‘ifru_netmask’ has incomplete type
   struct sockaddr ifru_netmask;
                   ^~~~~~~~~~~~
/usr/include/linux/if.h:238:20: error: field ‘ifru_hwaddr’ has incomplete type
   struct  sockaddr ifru_hwaddr;
                    ^~~~~~~~~~~

Check usable header "linux/if.h": NO

meson.build:36:4: ERROR:  Assert failed: Could not find <linux/if.h>
```
The following versions are installed:
```
gcc:
  Installed: 4:6.3.0-4
  Candidate: 4:6.3.0-4
  Version table:
 *** 4:6.3.0-4 500
        500 http://ftp.de.debian.org/debian stretch/main amd64 Packages
        100 /var/lib/dpkg/status
libc6-dev:
  Installed: 2.24-11+deb9u3
  Candidate: 2.24-11+deb9u3
  Version table:
 *** 2.24-11+deb9u3 500
        500 http://ftp.de.debian.org/debian stretch/main amd64 Packages
        100 /var/lib/dpkg/status
linux-libc-dev:
  Installed: 4.9.110-1
  Candidate: 4.9.110-1
  Version table:
 *** 4.9.110-1 500
        500 http://ftp.de.debian.org/debian stretch/main amd64 Packages
        100 /var/lib/dpkg/status
```

```
ninja: Entering directory `_build/'
[1/6] Compiling C++ object 'cppd@exe/cppmain.cpp.o'.
[2/6] Compiling D object 'dcpp@exe/dmain.d.o'.
[3/6] Compiling C++ object 'dcpp@exe/libfile.cpp.o'.
[4/6] Linking target dcpp.
FAILED: dcpp»
gdc  -o dcpp 'dcpp@exe/dmain.d.o' 'dcpp@exe/libfile.cpp.o'»»»
dcpp@exe/libfile.cpp.o: In function `print_hello(int)':
/tmp/meson-0.47.1/test cases/d/10 d cpp/_build/../libfile.cpp:4: undefined reference to `std::cout'
/tmp/meson-0.47.1/test cases/d/10 d cpp/_build/../libfile.cpp:4: undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::operator<< <std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*)'
/tmp/meson-0.47.1/test cases/d/10 d cpp/_build/../libfile.cpp:4: undefined reference to `std::ostream::operator<<(int)'
/tmp/meson-0.47.1/test cases/d/10 d cpp/_build/../libfile.cpp:4: undefined reference to `std::basic_ostream<char, std::char_traits<char> >& std::operator<< <std::char_traits<char> >(std::basic_ostream<char, std::char_traits<char> >&, char const*)'
dcpp@exe/libfile.cpp.o: In function `__static_initialization_and_destruction_0(int, int)':
/usr/include/c++/6/iostream:74: undefined reference to `std::ios_base::Init::Init()'
/usr/include/c++/6/iostream:74: undefined reference to `std::ios_base::Init::~Init()'
collect2: error: ld returned 1 exit status
[5/6] Compiling D object 'cppd@exe/libfile.d.o'.
ninja: build stopped: subcommand failed.
```

--- a/test cases/d/10 d cpp/libfile.d
+++ b/test cases/d/10 d cpp/libfile.d
@@ -1,5 +1,5 @@
-import std.stdio;
+import core.stdc.stdio;
 
 extern (C++) void print_hello(int i) {
-    writefln("Hello. Here is a number printed with D: %d", i);
+    printf("Hello. Here is a number printed with D: %d\n", i);
 }

